bench.go

```diff
@@ -127,14 +127,17 @@ type result struct {
 // connectionBenchmark performs a system benchmark by issuing requests at a
 // specified rate and capturing the latency distribution.
 type connectionBenchmark struct {
-	requester            Requester
-	requestRate          uint64
-	duration             time.Duration
-	expectedInterval     time.Duration
-	histogram            *hdrhistogram.Histogram
-	uncorrectedHistogram *hdrhistogram.Histogram
-	totalRequests        uint64
-	elapsed              time.Duration
+	requester          Requester
+	requestRate        uint64
+	duration           time.Duration
+	expectedInterval   time.Duration
+	successHistogram   *hdrhistogram.Histogram
+	unSuccessHistogram *hdrhistogram.Histogram
```

Comment:
Sorry if this is overly pedantic, but `unSuccessHistogram` might be confusing to people since they might think it means unsuccessful requests. Is there a better name? `uncorrected...` seems the most clear, though I know it's a bit long.